extensions/2.0/Khronos/KHR_lights/README.md

```diff
@@ -0,0 +1,151 @@
+
+```javascript
+"scenes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+## Light Types
+
+All light types share the common set of properties listed below.
+
+### Light Shared Properties
+
+| Property | Description | Required |
+|:-----------------------|:------------------------------------------| :--------------------------|
+| `name` | Name of the light. | No, Default: `""` |
+| `color` | RGB value for light's color in linear space. | No, Default: `[1.0, 1.0, 1.0]` |
+| `intensity` | Brightness of light in. The units that this is defined in depend on the type of light. `point` and `spot` lights use luminous intensity in candela (lm/sr) while `directional` and `ambient` lights use illuminance in lux (lm/m^2) | No, Default: `1.0` |
+| `type` | Declares the type of the light. | :white_check_mark: Yes |
+| `range` | Hint defining a distance cutoff at which the light's intensity may be considered to have reached zero. Supported only for `point` and `spot` lights. | No, Default: `0.0` |
+
+### Ambient
+
+Ambient lights define constant lighting throughout the scene. They are referenced only by a scene object and only 1 can be referenced per scene. It's intensity is defined in lumens per metre squared, or lux (lm/m^2).
+
+### Directional
+
+Directional lights are light sources that act as though they are infinitely far away and emit light in the direction of the local +z axis. This light type inherits the orientation of the node that it belongs to but ignores position and scale. Because it is at an infinite distance, the light is not attenuated. Its intensity is defined in lumens per metre squared, or lux (lm/m^2).
+
+### Point
+
+Point lights emit light in all directions from its position in space (and ignore rotation and scale). The brightness of the light attenuates in a physically correct manner as distance increases from the light's position (i.e. brightness goes like the inverse square of the distance). Point light intensity is defined in candela, which is lumens per square radian (lm/sr).
+
+### Spot
+
+Spot lights emit light in a cone in the direction of the local +z axis. The angle and falloff of the cone is defined using two numbers, the `innerConeAngle` and `outerConeAngle`. As with point lights, the brightness also attenuates in a physically correct manner as distance increases from the light's position (i.e. brightness goes like the inverse square of the distance). Spot light intensity refers to the brightness inside the `innerConeAngle` and is defined in candela, which is lumens per square radian (lm/sr). Engines that don't support two angles for spotlights should use `outerConeAngle` as the spotlight angle (leaving `innerConeAngle` to implicitly be `0`).
```

Comment:
> Engines that don't support two angles for spotlights should use `outerConeAngle` as the spotlight angle (leaving `innerConeAngle` to implicitly be `0`).

Just noticed this. Are we okay with this? I would think this will cause inconsistent behavior between clients.

Reply:
I forget the reason for this... was there a key engine that doesn't support both?

Reply:
Unity only support one angle on spot lights:
![image](https://user-images.githubusercontent.com/19158740/40563176-b95f7d2a-6018-11e8-9f4a-cd10b2ffb618.png)

Reply:
Hm I was looking at [SpotLight in the scripting API](https://docs.unity3d.com/2018.1/Documentation/ScriptReference/Experimental.GlobalIllumination.SpotLight.html) suggesting it has both `coneAngle` and `innerConeAngle`? 😕

Reply:
Looks like that was just added in 2018.1 and is experimental (it doesn't exist in 2017 or older).